contracts/_utils/SafeMathCompatibility.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+// NOTE: Adding an intermediate library to support older version of safemath.
```

Comment:
Thinking of this as an intermediary, then replacing with the more basic math ops later?

Reply:
yes